packages/build-tools/src/builders/android.ts

```diff
@@ -11,7 +11,32 @@ import { restoreCredentials } from '../android/credentials';
 import { configureBuildGradle } from '../android/gradleConfig';
 import { prebuildAsync } from '../utils/prebuild';
 
-export default async function androidBuilder(ctx: BuildContext<Android.Job>): Promise<string[]> {
+export default async function androidBuilder(ctx: BuildContext<Android.Job>): Promise<string> {
+  let isBuildSuccess = true;
+  try {
```

Comment:
It's almost the same code as for ios, but I don't think there is a need to extract that logic yet